### ISSUE TITLE
Importing `imports/server/XXX.js` from `server/main.js` causes an ESLint error on Windows

### ISSUE THREAD

**Issue by exKAZUu** [clayne11/eslint-import-resolver-meteor#13]
I assume that Windows-specific directory delimiter `\` causes this problem.
I created a reproduction repository. I got the following errors.
https://github.com/exKAZUu/test-eslint-import-resolver-meteor

```
C:\Users\exKAZUu\Desktop\test-eslint-import-resolver-meteor>npm run eslint

> test-eslint-import-resolver-meteor@ eslint C:\Users\exKAZUu\Desktop\test-eslint-import-resolver-meteor
> eslint . --fix

The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.

C:\Users\exKAZUu\Desktop\test-eslint-import-resolver-meteor\server\main.js
  3:32  error  Unable to resolve path to module '../imports/server/value34'  import/no-unresolved

✖ 1 problem (1 error, 0 warnings)
```

**Comment by clayne11** [comment]
Likely due to the `\`. Are you willing to take a crack at fixing it? I don't use Windows so this is a pain for me to fix/maintain.

**Comment by exKAZUu** [comment]
Okay, I will try it today :+1:

**Comment by clayne11** [comment]
FYI you shouldn't really be doing `../imports` - you can simply use `/imports`.

**Comment by exKAZUu** [comment]
I see. Thank you for sharing it.

**Comment by clayne11** [comment]
Thanks for the PR!

Published as [v0.3.2](https://github.com/clayne11/eslint-import-resolver-meteor/releases/tag/v0.3.2).

**Comment by exKAZUu** [comment]
Sorry. My fix was wrong. I will make another PR ASAP.
